### PR TITLE
Addition of simple component, router, and service subgenerators

### PR DESCRIPTION
**Pull request by orangedrink**
This is going to do the job for my own project I believe but let me know if you have feedback.

### CODE REVIEW

**Comment by victorsosa** [comment]
Very nice job, men; I will doble check later but for now you need to add some doc to the readme file :+1:  nice men.

File: package.json [hunk 1]
```diff
@@ -21,7 +22,7 @@
     "yeoman-generator"
   ],
   "dependencies": {
-    "angular2-seed-advanced":"1.0.0",
+    "angular2-seed-advanced":"angular2-seed-advanced",
```
Comment (victorsosa) [inline package.json]: please keep this as 1.0.0; I will push the package to npm after this patch

**Comment by victorsosa** [comment]
Thanks, nice job

**Comment by orangedrink** [comment]
Yay! Thanks. That took me way too many commits. Heh.

**Comment by orangedrink** [comment]
Thanks for your patience with me while I learn.

**Comment by victorsosa** [comment]
No problema, don't worry about the commits, any contribution is welcome

**Comment by victorsosa** [comment]
the npm was published so now you we can use npm to install the packages.

Remember to do:
1. npm uninstall package
2. npm install package

note: this only because you used the npm link in development, after this you don't need npm link anymore; because you are point to the released package.

https://www.npmjs.com/search?q=generator-ng2-seed-advanced
https://www.npmjs.com/package/angular2-seed-advanced

**Pull request merged**